docs/getting-started/kernel-upgrade.md

```diff
@@ -46,12 +46,12 @@ sudo reboot
 uname -r
 ```
 
-(DEBIAN ONLY): If you would like to upgrade to the latest Kernel (AGGRESSIVE UPGRADE), follow the commands below:
+(Debian ONLY): If you would like to upgrade to the latest Kernel (AGGRESSIVE UPGRADE), follow the commands below:
 
 > **Warning**
-> The latest Kernel that Debian officially supports is available in the `unstable release`. Debian Unstable (also known by its codename "Sid") is not strictly a release, but rather a rolling development version of the Debian distribution containing the latest packages that have been introduced into Debian. Upgrading to the latest Kernel might potentially introduce breaking changes to your system, so please do at your own risk.
+> The latest Kernel that Debian officially supports is available in the `unstable release`. Debian Unstable (also known by its codename "SID") is not strictly a release, but rather a rolling development version of the Debian distribution containing the latest packages that have been introduced into Debian. Upgrading to the latest Kernel might potentially introduce breaking changes to your system, so please do at your own risk.
```

Comment:
Why not 
```
**Warning**: xxxxx
```

Reply:
This is a good catch, I will fix it in a sec.

Reply:
Working like a charm - https://github.com/yqlbu/dae/blob/ci-markdown-lint/docs/getting-started/kernel-upgrade.md